GCTC-NTGC/gc-digital-talent
# Admin - Users table view column, email value

Resolves #4053.

## Summary
On the Admin Users table, adds a **View** column with a value that links to user profile, similar to Edit and makes href value for **Email** value mailto: instead of linking to user profile.

## Testing
1. Compile languages `npm run intl-compile --workspace="admin"`
2. Build admin `npm run production --workspace="admin"`
3. Navigate to /en/admin/users
4. Confirm the View value links to user profile
5. Confirm the Email value links to a corresponding email address
6. Switch to French
7. Confirm the Voir value links to user profile
8. Confirm the Courriel value links to a corresponding email address
